### ISSUE TITLE
New Update Compatibility? 

### ISSUE THREAD

**Issue by RadonSound** [GaijinEntertainment/fmod_studio_warthunder_for_modders#196]
Is it already known if the [Sky Guardians Fmod project update] is compatible with the new major patch?

**Comment by Uchuujin01** [comment]
What I can say for sure, interior and exterior weapon sound events will play simultaneously for the gunner perspective. Because we got multiply requests for that from sound modders. So you will be able to quick switch from 3rd to 1st person and back, and hear proper sound,

**Comment by Uchuujin01** [comment]
I don't think it is much of a change, so, yes, it will be compatible.

**Comment by IWSM** [comment]
> What I can say for sure, interior and exterior weapon sound events will play simultaneously for the gunner perspective. Because we got multiply requests for that from sound modders. So you will be able to quick switch from 3rd to 1st person and back, and hear proper sound,

won't that impact the performance of Fmod and the sound engine too much?

**Comment by Uchuujin01** [comment]
> won't that impact the performance of Fmod and the sound engine too much?

No, I've measured performance along with those changes, it works well. Also, we have a similar mechanism in aircraft weapons.
